go.mod

```diff
@@ -9,100 +9,353 @@ module k8s.io/kubernetes
 go 1.16
```

Comment:
the drastic changes in this file are unexpected... you might need to reset and redo the pin-dependency.sh / update-vendor.sh changes

Reply:
Reset is done.

/skip 
for pull-kubernetes-e2e-aks-engine-windows-containerd